pymoliere/construct/dask_process_global.py

```diff
@@ -69,10 +69,16 @@ def safe_get_worker():
   except:
     return LOCAL_MOCK_WORKER
 
-def add_global_preloader(client:Client, preloader:WorkerPreloader)->None:
+def add_global_preloader(
```

Comment:
I'm safe to change this function because its only current usages are keyword-augment only.